entrypoint.sh

```diff
@@ -9,7 +9,7 @@ set -e
 function run_tfupdate {
   case ${INPUT_RESOURCE} in
     terraform)
-      VERSION=$(tfupdate release latest hashicorp/terraform)
+      VERSION=$(tfupdate release list hashicorp/terraform| grep -v ".*-rc[0-9]$"| sort -r| head -1)
```

Comment:
Sorry, maybe this should be okay for now, but `sort -r -n` would be better if we do numeric sorting. 🤔

Reply:
I'm sorry. `-n` doesn't work in some cases. please ignore 🙇🏼‍♂️ 

```
echo "0.12.17
0.12.18
0.12.19
0.12.20
0.12.21
0.1.30
0.12.5" | grep -v ".*-rc[0-9]$"| sort -r -n

0.12.5
0.12.21
0.12.20
0.12.19
0.12.18
0.12.17
0.1.30
```

Reply:
Seems `sort --version-sort -r` works.